test/suites/dev/test-precompile/test-precompile-conviction-voting.ts

```diff
@@ -1,14 +1,42 @@
 import "@moonbeam-network/api-augment";
-import { beforeAll, beforeEach, describeSuite, expect, fetchCompiledContract } from "@moonwall/cli";
+import {
+  DevModeContext,
+  beforeAll,
+  beforeEach,
+  describeSuite,
+  expect,
+  fetchCompiledContract,
+} from "@moonwall/cli";
 import {
   ALITH_ADDRESS,
+  ETHAN_ADDRESS,
+  ETHAN_PRIVATE_KEY,
   PRECOMPILE_CONVICTION_VOTING_ADDRESS,
   createViemTransaction,
 } from "@moonwall/util";
 import { Abi, decodeEventLog, encodeFunctionData } from "viem";
 import { expectEVMResult, extractRevertReason } from "../../../helpers/eth-transactions.js";
 import { expectSubstrateEvent } from "../../../helpers/expect.js";
-import { createProposal } from "../../../helpers/voting.js";
+import { cancelProposal, createProposal } from "../../../helpers/voting.js";
+
+async function voteYes(
+  context: DevModeContext,
+  convictionVotingAbi: Abi,
+  proposalIndex: number,
+  amount: bigint,
+  conviction: number
+) {
+  const rawTx = await createViemTransaction(context, {
```

Comment:
Did you try this with the writePrecompile() fn instead? e.g.
```
 const rawTxn = await context.writePrecompile!({
        precompileName: "ConvictionVoting",
        functionName: "voteYes",
        args: [proposalIndex, amount, conviction],
        rawTxOnly: true,
      })
```

I prefer to use the wrapper as it improve legibility. I know you were extending what was written before but i think we should refactor the file over to the new style as its far less boilerplate

Reply:
done